### DISCUSSION TITLE
Get start with UNO

### DISCUSSION THREAD

**Discussion by Dvij30** [unoplatform/uno#13665] [General]
Hey I recently attended the UNO workshop in Concordia University. 
I would like to contribute to UNO platform so it would be much appreciated if any of you can guide me how to contribute to UNO platform and start solving the good first issues.

**Comment by limeniye** [comment]
Hello, @Dvij30.
Thank you for interesting of Uno contribute.
The Uno team is always happy to welcome new contributers.
I recommend you to check the following 4 links:

[CONTRIBUTING](https://github.com/unoplatform/uno/blob/master/CONTRIBUTING.md)

[Epic issue](https://github.com/unoplatform/uno/issues/12893) 

[MartinZikmund](https://youtube.com/playlist?list=PLC0wF4Qw6SDNu61eMpdGCw-c5RR2Tk2R6&si=ziqG1rlzn9ncuPM_)

[jeromelaban](https://youtube.com/playlist?list=PLl_OlDcUya9pRqt9A0vpyi89Vl-uq3nSb&si=I1MG_wBnJgbBRYj0)

**Reply by Dvij30** [reply]
Thank you for considering me and for providing me the links. The links were very useful to be for getting bit more familiar with the environment I will try my best in solving the good first issue and would love to receive feedback for improvement also.
